core/primitives/src/trie_key.rs

```diff
@@ -83,6 +83,7 @@ pub mod col {
         (PENDING_DATA_COUNT, "PendingDataCount"),
         (POSTPONED_RECEIPT, "PostponedReceipt"),
         (DELAYED_RECEIPT_OR_INDICES, "DelayedReceiptOrIndices"),
+        (u8::MAX, "unused"),
```

Comment:
I'm not super comfortable adding this entry here as by definition this array is perhaps supposed to contain all keys that are actually used. Please do revert this before merging PR.

For our use case, we can perhaps convert the entries of this array to a hashmap to retrieve key name and panic if key isn't from this hashmap.

Reply:
Hmm I see

I could make a copy of this array and revert the original to what it was. Or make a `fn` that would skip over the missing entries. There's no way to derive a `const` hashmap from this right? 

I'd be great to hear also others' opinions

Reply:
I meant we can make a hashmap within the test where we are using the mapping, not here in trie_key. Something like `let name_map = ALL_COLUMNS_WITH_NAMES.iter().cloned().collect::<HashMap<_, _>>();`

Reply:
It's bad to have a hole, it's bad to use u8::max, it's bad to have unused entry... 
I have no strong feelings either way, I guess I dislike all options equally ;) 

Looking at current usages it's totally fine, it's merely used for error messages and stats like you said. Perhaps it's less future proof.

Reply:
I'll just use the column identifier for now in all places, it's a non issue to understand what it means if there's an error.